Release/7.0 minor fts updates (#3021)
couchbase/docs-server: modules/fts/pages/fts-introduction.adoc

* xref:fts-supported-queries-wildcard.adoc[Wildcard]
* xref:fts-supported-queries-boosting-the-score-query.adoc[Boosting the Score]

> Full Text Search includes pre-built _text analyzers_ including but not limited to the following languages: Arabic, CJK characters (Chinese, Japanese, and Korean), English, French, Hindi, Italian, Kurdish, Persian, and Portuguese.  For the current list of all supported languages in the Couchbase Server refer to: xref:fts-index-analyzers.adoc#Supported-Languages[Supported Languages]

Reviewer: ```suggestion
Full Text Search includes pre-built text analyzers for multiple languages. For the current list of all supported languages in Couchbase Server refer to: xref:fts-index-analyzers.adoc#Supported-Languages[Supported Analyzer Languages]
```
